deployment/v3/mosip/mosip-file-server/values.yaml

```diff
@@ -0,0 +1,15 @@
+image:
+  registry: docker.io
+  repository: mosipdev/mosip-file-server
+  tag: develop
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: Always
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
```

Comment:
please remove values.yaml file as it is not required here

Reply:
removed the file.